.github/workflows/download-crowdin-generic.yml

```diff
@@ -0,0 +1,75 @@
+name: Generic Crowdin download Action
+
+on:
+  workflow_call:
+    inputs:
+      PROJECT_VERSION:
+        description: Project version
+        required: true
+        type: string
+      PROJECT_BRANCH:
+        description: Project git branch 
+        required: false
+        type: string
+    secrets:
+      CROWDIN_GITHUB_TOKEN:
+        required: true
+      CROWDIN_PROJECT_ID:
+        required: true
+      CROWDIN_PERSONAL_TOKEN:
+        required: true
+      SWF_GPG_PRIVATE_KEY:
+        required: true
+      SWF_GPG_PASSPHRASE:
+        required: true
+
+jobs:
+
+  crowdin-download:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.CROWDIN_GITHUB_TOKEN }}
+          ref: ${{ inputs.PROJECT_BRANCH }}
+      - name: Reset translation branch
+        run: |
+          git push origin origin/${{inputs.PROJECT_BRANCH}}:refs/heads/integration/${{inputs.PROJECT_VERSION}}-translation -f
+      - name: Crowdin push
+        uses: crowdin/github-action@v2
+        with:
+          upload_sources: false
+          upload_translations: false
+          download_translations: true
+          localization_branch_name: integration/${{inputs.PROJECT_VERSION}}-translation
```

Comment:
crowdin_branch_name: ${{inputs.CROWDIN_MAINTENANCE_BRANCH_VERSION}}
is a key param. It is not put for develop as it is the main branch and not needed. But it is used for develop-exo, develop-meed and 6.5.x

Reply:
I will take care of it. Thanks

Reply:
For the crowdin branch param I've checked this https://github.com/crowdin/github-action/blob/4eb602e9ff92bc79d92226486ddc39ddc15c83b7/entrypoint.sh#L306 It's OK to put an empty string

Reply:
yes that 's for the main branch not the others

Reply:
We start to use branches in crowdin project

Reply:
PR updated